src/Apps/Artist/Routes/AuctionResults/SingleAuctionResultPage/AuctionResult.tsx

```diff
@@ -18,11 +18,14 @@ interface AuctionResultProps {
 export const AuctionResult: React.FC<AuctionResultProps> = ({
   auctionResult,
 }) => {
-  const { router } = useRouter()
+  const router = useRouter()
   const { comparableAuctionResults, title, artist, internalID } = auctionResult
 
   const results = extractNodes(comparableAuctionResults)
 
+  const returnTo = router.match?.location?.query?.returnTo
```

Comment:
So after your last comment we don't need `returnTo` I believe?

Reply:
It sounds to me like design should arbitrate how this should behave. If we wanted to preserve the status quo it's a bit more complicated than just this because we need the title that it's returning to as well. "Back" just isn't going to cut it. Ask design, if it's going to take a while, remove this or use a `Breadcrumb` so it feels like it's not a back button.